### PR TITLE
[master < T0800-MG] Add microseconds precision to temporal types now functions

### PR DESCRIPTION
**Pull request by kostasrim**
[master < Task] PR
- [ ] Check, and update documentation if necessary
- [ ] Update [changelog](https://docs.memgraph.com/memgraph/changelog)

Don't merge this unless I update(tick) the update doc above.

### CODE REVIEW

File: src/utils/temporal.cpp [hunk 1]
```diff
@@ -79,36 +79,15 @@ Date::Date(const DateParameters &date_parameters) {
...
-}
+LocalTime UtcLocalTime() { return UtcLocalDateTime().local_time; }
 
 LocalDateTime UtcLocalDateTime() {
-  const auto utc_today = GetUtcFromSystemClockOrThrow();
-  return LocalDateTime({TMYearToUtcYear(utc_today.tm_year), TMMonthToUtcMonth(utc_today.tm_mon), utc_today.tm_mday},
-                       {utc_today.tm_hour, utc_today.tm_min, utc_today.tm_sec});
+  namespace chrono = std::chrono;
+  using TpMicros = chrono::time_point<chrono::system_clock, chrono::microseconds>;
+  TpMicros ts = chrono::time_point_cast<chrono::microseconds>(chrono::system_clock::now());
```
Comment (antaljanosbenjamin) [inline src/utils/temporal.cpp:89]:
```suggestion
  auto ts = chrono::time_point_cast<chrono::microseconds>(chrono::system_clock::now());
```

**Review by antaljanosbenjamin — CHANGES_REQUESTED** [review]
Is it possible to write/extend some of the tests? If that is hard (because it is very time dependent), then it doesn't worth the effort.

**Review by jbajic — APPROVED** [review]

File: src/utils/temporal.cpp (hunk 1 above)
Comment (kostasrim) [reply src/utils/temporal.cpp:89]: hahaha, :eyes: how did I miss that :D :D +100

Comment (kostasrim) [reply src/utils/temporal.cpp:89]: I will check for the tests and update

**Review by antonio2368 — APPROVED** [review]

**Comment by antonio2368** [comment]
Rebase this so we can merge it next

**Pull request closed without merging**